web/src/client/http.js

```diff
@@ -142,27 +142,36 @@ class HTTPClient {
       },
     });
 
-    try {
-      return await response.json();
-    } catch (e) {
-      console.warn("Expecting a JSON response", e);
-      return response.status === 200;
-    }
+    return response;
+  }
+
+  /**
+   * @param {string} url - Endpoint URL (e.g., "/l10n/config").
+   * @return {Promise<Response>} Server response.
+   */
+  async delete(url) {
+    const response = await fetch(`${this.baseUrl}/${url}`, {
+      method: "DELETE",
+    });
+
+    return response;
```

Comment:
I am not sure whether we should return the at all. At some point we could wrap the standard responses objects with our own information.

In any case, if we want to, we should do that in all methods to be consistent.

Reply:
well, response is important if we want to check if call succeed or failed ( method `ok` on response or if response is added as it is optional for non-get calls). What I do not like even on current `get` is that it returns `true/false` if response is not json, which is quite bad API. So providing whole response object is more clear for me.